src/main/java/frc/robot/Robot.java

```diff
@@ -51,7 +52,9 @@ public void robotPeriodic() {
   public void disabledInit() {}
 
   @Override
-  public void disabledPeriodic() {}
+  public void disabledPeriodic() {
+    SwerveDrive.navxGyro.reset();
```

Comment:
You may want to change this since the time between auto and teleop is disabled, so it will reset between the two

Reply:
@cwedin78 this may still be a problem